### ISSUE TITLE
Request a VSA 508 spot check review for AMA/Decision Review Part 2

### ISSUE THREAD

**Issue by MickinSahni** [department-of-veterans-affairs/va.gov-team#7339]
### For reference

**VSA Development Review** - [https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review)

## Team Name

Public Websites

## Product/Feature Name

AMA/Decision Review Part 2

## Product description

From https://github.com/department-of-veterans-affairs/va.gov-team/issues/6421#issue-574929522

> As a Veteran, I would like to submit all new appeals under the decision review process.
> As a Veteran, I would like to use the VA.gov website to find content related to both the decision-review and legacy-appeal processes.
> As a content writer, I would like to balance plain language [appeal] with the Client’s desire for adoption of decision review terminology.

## Review stage

~**Development intent review** (Review design to discuss approach to take with development, identify what components and semantic HTML would be appropriate, identify any accessibility issues and how to address) - This would happen as designs are passed to development, before development begins.~

- [x] **Spot check review** (Review and follow the steps in the [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md) first.) This happens well before launch, so that the 508 office can review and any issues addressed *before* launch.

**Drupal preview links:**

1. - [ ] [Decision reviews and appeals](https://preview.prod.vfs.va.gov/preview?nodeId=3071)
2. - [ ] [After you request a decision review](https://preview-prod.vfs.va.gov/preview?nodeId=3008) 
3. - [ ] [Supplemental Claims](https://preview-prod.vfs.va.gov/preview?nodeId=3011)
4. - [ ] [Higher-Level Reviews](https://preview-prod.vfs.va.gov/preview?nodeId=3003)
5. - [ ] [Board Appeals](https://preview-prod.vfs.va.gov/preview?nodeId=3029)
6. - [ ] [After a Board Appeal](https://preview-prod.vfs.va.gov/preview?nodeId=3013)
7. - [ ] [Contested Claims](https://preview-prod.vfs.va.gov/preview?nodeId=3030)
8. - [ ] [Fiduciary claims](https://preview-prod.vfs.va.gov/preview?nodeId=3016)
9. - [ ] [Insurance claims](https://preview-prod.vfs.va.gov/preview?nodeId=3025)
10. - [ ] [FAQs](https://preview-prod.vfs.va.gov/preview?nodeId=3004) 
11. - [ ] [Get help with review request ](https://preview-prod.vfs.va.gov/preview?nodeId=3009)
12. - [ ] [Board hearings with a VLJ](https://preview-prod.vfs.va.gov/preview?nodeId=3010)

 

- [ ] **Other:** Please indicate what you're requesting

## Point of contact

Mickin

## Preferred times for dev intent checkpoint

n/a

## Staging URL and credentials

https://github.com/department-of-veterans-affairs/va.gov-team/issues/7153#issue-587037392

## Test plan

https://github.com/department-of-veterans-affairs/va.gov-team/issues/6421#issuecomment-604175380

## Previously shared accessibility/508 feedback
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/6335

**Comment by lwalsh617** [comment]
I added Drupal preview links to the top of the ticket.

**Comment by jenstrickland** [comment]
@lwalsh617 Hola! I tried entering the url in the screenshot above, but it doesn't seem to be a viable url (I added https:// to the beginning).  Would you have the actual urls available, by chance?

**Comment by MickinSahni** [comment]
@jenstrickland hey jennifer, the timing of this and staging review seem to colliding. would it make sense to roll this into the staging review request?

**Comment by jenstrickland** [comment]
@MickinSahni The Staging Review is less intensive than the spot check (which is intended to help make sure nothing shows up in the staging review). 

I tried to start the spot check today, but don't have the urls to access. Might you have live urls at the ready?

**Comment by lwalsh617** [comment]
Hey @RLHecht - Do you know the answer to @jenstrickland's question above? She's unable to access the Drupal preview links at the top of this ticket. Thank you!

**Comment by MickinSahni** [comment]
> @MickinSahni The Staging Review is less intensive than the spot check (which is intended to help make sure nothing shows up in the staging review).
> 
> I tried to start the spot check today, but don't have the urls to access. Might you have live urls at the ready?

Ok, gotcha. I think in that case I might just open up the staging review sooner rather than later. Thoughts? 
 
Hmm Laura would be the best bet for urls. Would the urls here not be accurate?
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6421

Also, @lwalsh617 - given we're in accessibility spot-check and pending a "staging review" i'm not sure the Friday launch date is viable. We can talk about it in grooming. I don't think there's a particular date associated with this AMA Part 2.

**Comment by RLHecht** [comment]
@lwalsh617 The links you have up top can't be clicked through, so that might be the issue. Though that first preview link for the hub page looks like it's written wrong. It should be preview-prod.vfs.va.gov/preview?nodeId=3071

**Comment by jenstrickland** [comment]
Adding http:// or https:// to the start of that link doesn't work either. I tried typing the urls in, with the https:// and it doesn't seem to be valid.

**Comment by RLHecht** [comment]
Are you doing this in the VA network? Drupal doesn't work otherwise.

**Comment by jenstrickland** [comment]
Thanks, @RLHecht, that helps. It is complicated though because I can't use the axe browser tools to check. I did check the page you provided in your last message and there's an accessibility issue with the sidebar. There's a message at the top of the page from the build process.

**Comment by RLHecht** [comment]
@jenstrickland That right rail is the standard VBA one. It should be the same on header level as all the other hubs.

**Comment by jenstrickland** [comment]
Yep, I'd love to see it addressed — but it is a platform-level design system thing. We'll need to create an issue, aim to have the design system issues addressed.

**Comment by MickinSahni** [comment]
1. [Decision reviews and appeals](https://preview.prod.vfs.va.gov/preview?nodeId=3071)
 2. [After you request a decision review](https://preview-prod.vfs.va.gov/preview?nodeId=3008)
 3. [Supplemental Claims](https://preview-prod.vfs.va.gov/preview?nodeId=3011)
4. [Higher-Level Reviews](https://preview-prod.vfs.va.gov/preview?nodeId=3003)

@lwalsh617 this work?

**Comment by lwalsh617** [comment]
@jenstrickland All 12 Drupal preview links are now working and are at the top of this ticket.

**Comment by jenstrickland** [comment]
@lwalsh617 It looks like something broke the 12 urls above.

**Comment by RLHecht** [comment]
@jenstrickland @lwalsh617 These work for me if you have it as http and not https. Though I must have written the wrong link down AGAIN for that hub landing page. All Drupal links should begin: preview-prod.vfs.va.gov/preview?nodeId= and then whatever the node ID number is. But you won't be able to click through these if you're not doing it within the VA network. 
<img width="785" alt="Screen Shot 2020-03-31 at 5 51 09 PM" src="https://user-images.githubusercontent.com/48728214/78078447-3626be80-7378-11ea-81c4-8338a5c19efe.png">

**Comment by jenstrickland** [comment]
The list of links in the request are fixed now. I have them all copied down in a text file. 

I also started the spot check / staging a11y review. Most of the issues are template issues, with the exception of the Supplemental Claims page, where there are content issues.

Here are the **results for the first four pages**:

1. Decision reviews and appeals — https://preview-prod.vfs.va.gov_preview_nodeid=3071

    **Template Issue |** Heading levels should only increase by one
    Section:nth-child(1) > h4
    `<h4>Message us</h4>`

   **Template Issue |**  All page content must be contained by landmarks
    `<div class="veteran-banner">`



2. After you request a decision review — https://preview-prod.vfs.va.gov/preview?nodeId=3008

    **Template Issue |** Heading levels should only increase by one
    `<h4 class="va-footer-linkgroup-title>Veteran programs and services</h4>`

    **Template Issue |** Ensure landmarks are unique
    `<div id="vetnav" role="navigation">`
    And #va-detailpage-sidebar
    [strange, because I don't **see** a sidebar on this page]

    **Template Issue |** All page content must be contained by landmarks
    .vads-u-padding-x--1
    `<div class="vads-u-padding-x--1 medium-screen:vads-u-padding-x--2">`



3. Supplemental Claims — https://preview-prod.vfs.va.gov/preview?nodeId=3011

    Heading levels should only increase by one
    `<h3 id="can-i-file-a-supplemental-clai">`

    **Template Issue |** Heading levels should only increase by one
    `<h4 class="va-footer-linkgroup-title>Veteran programs and services</h4>`

    **Template Issue |** Ensure landmarks are unique
    `<div id="vetnav" role="navigation">`
    And #va-detailpage-sidebar
    [strange, because I don't **see** a sidebar on this page]

    **Template Issue |** All page content must be contained by landmarks
    .vads-u-padding-x--1
    `<div class="vads-u-padding-x--1 medium-screen:vads-u-padding-x--2">`

    Within the subway steps content, for the text that is currently `<p> <strong>Words</strong> … </p>`, these should be headings, since the screen reader does not indicate they are headings, and it isn't clear when it is read out — it sounds like a sentence isn't finished.

    For the "subheadings" By mail, Compensation, In person, and By fax:
    These should be a list with headings so the screen reader may provide the context. The way this is read by a screen reader, it sounds like the Veteran must complete the form by mail, as the context of options is not read. Compensation isn't clear as the type of benefit applied for. It might be better if instead of "Compensation" it read "Address to send Compensation claims to".

    For all of the state abbreviations, they must be coded as abbreviations, providing the state name in the title.
    If you live in AK, AZ, CA, CO, HI, ID, IA, KS, MN, MT, NB, NV, NM, ND, OK, OR, SD, TX, UT, WA, WY, etc…
    `<abbr title="Alaska">AK</abbr>, <abbr title="Arizona">Arizona</abbr>`, etcetera
    At present, it is confusing because it reads phonetically instead.



4. Higher-Level Reviews — https://preview-prod.vfs.va.gov/preview?nodeId=3003

    **Template Issue |** Heading levels should only increase by one
    `<h4 class="va-footer-linkgroup-title>Veteran programs and services</h4>`

    **Template Issue |** Ensure landmarks are unique
    `<div id="vetnav" role="navigation">`
    And #va-detailpage-sidebar
    [strange, because I don't **see** a sidebar on this page]

    **Template Issue |** All page content must be contained by landmarks
    .vads-u-padding-x--1
    `<div class="vads-u-padding-x--1 medium-screen:vads-u-padding-x--2">`

**Comment by jenstrickland** [comment]
@MickinSahni All the pages have been audited. I can add these as tickets to the Staging Review, if you like.
